### PR TITLE
test: Change the configuration toml

### LINKED ISSUE #2675: add a test where we change the machine type for an invalid one

### PR DESCRIPTION
**Pull request by GabyCT**
This PR is a test where we change the machine type at the configuration toml
for an invalid one and we verify that is not possible to run a container.

Fixes #2675

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/change_configuration_toml/change_configuration_toml.bats [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+setup() {
+	clean_env
+	run check_processes
+	echo "$output"
+	cp -a "${default_kata_config}" "${CONFIG_FILE}"
+	sudo sed -i 's|^machine_type.*|machine_type = "foo"|g' "${default_kata_config}"
+}
+
+@test "modify configuration toml" {
```
Comment (devimc) [inline integration/change_configuration_toml/change_configuration_toml.bats:24]: maybe you should use a less generic name, what about "bad machine type" ?

File: integration/change_configuration_toml/change_configuration_toml.bats [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+tmp_data_dir="$(mktemp -d)"
+CONFIG_FILE="${tmp_data_dir}/configuration.toml"
+
+setup() {
+	clean_env
+	run check_processes
+	echo "$output"
+	cp -a "${default_kata_config}" "${CONFIG_FILE}"
+	sudo sed -i 's|^machine_type.*|machine_type = "foo"|g' "${default_kata_config}"
```
Comment (devimc) [inline integration/change_configuration_toml/change_configuration_toml.bats:21]: this should be part of the @ test

File: integration/change_configuration_toml/change_configuration_toml.bats [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+@test "modify configuration toml" {
+	run docker run -d --name "${container_name}" --runtime "${RUNTIME}" "${image}" sh -c "${payload}"
+	echo "$output"
+	[ "$status" -eq 0 ]
```
Comment (devimc) [inline integration/change_configuration_toml/change_configuration_toml.bats:27]: mmm the test should fail, so I think the condition should be `[ "$status" -ne 0 ]`

**Comment by GabyCT** [comment]
@devimc changes applied

**Pull request merged**